themes/bootstrap3/templates/ajax/resolverLink.phtml

```diff
@@ -1,5 +1,5 @@
 <?php if (!empty($link['href'])): ?>
-  <a href="<?=$this->escapeHtmlAttr($link['href'])?>" title="<?=$this->transEscAttr($link['service_type'] ?? '')?>"<?=!empty($link['access']) ? ' class="access-' . $link['access'] . '"' : ''?>>
+  <a href="<?=$this->escapeHtmlAttr($link['href'])?>" title="<?=$link['service_type'] ? $this->transEscAttr('ServiceType::' . $link['service_type']) : ''?>"<?=!empty($link['access']) ? ' class="access-' . $link['access'] . '"' : ''?>>
```

Comment:
If there's a possibility that service_type may be unset, I'd recommend adjusting to: `title="<?=!empty($link['service_type']) ? $this->transEscAttr('ServiceType::' . $link['service_type']) : ''?>"` to avoid the possibility of trying to access an unset value. (Doing a direct check against `$link['service_type']` will issue a notice if 'service_type' is missing, but using `empty` will avoid that potential problem).

Reply:
Thanks @demiankatz , using `empty()` now for checking uset value of `$link['service_type'] `